### PR TITLE
GAN_mnist: fix `generator` return doc

### PR DESCRIPTION
**Pull request by vertexcite**
Remove `logits` from return doc, since it isn't actually returned.

### CODE REVIEW

**Comment by mcleonard** [comment]
Thanks for catching this!
